content/05.proteolysis.md

```diff
@@ -8,13 +8,27 @@ Peptides that are too short are less likely to uniquely match to a single protei
 There are many choices of enzymes and chemicals that hydrolyze proteins into peptides. 
 This section summarizes potential choices and their strengths and weaknesses. 
 
-Trypsin is the most common choice of protease for proteome hydrolysis [@DOI:10.1002/mas.21376]. 
- 
+Trypsin is the most common choice of protease for proteome hydrolysis [@DOI:10.1002/mas.21376].
+Irrespective of the experimental strategies and methodologies used in proteomics, in-gel or in-solution bottom-up proteomics workflows uses enzymatic digestion for protein identification. 
+Depending on the amino-acid cleavage specificities, several enzymes are being used in shotgun proteomics approaches. 
+Nevertheless, trypsin remains the primary choice because of its specificity, availability, efficiency and is comparatively cheaper than other enzymes. 
+Trypsin cleaves at the C-terminus of basic amino acids, arginine and lysine. 
+Since the generated peptides are very short in length (3-35 amino acids), they can be subjected to chromatography separation, MS-based peptide fragmentation and database-enabled identification.
+But the drawback is that majority (56%) of the tryptic peptides are very small in length (≤ 6 amino acids) and hence it’s very difficult for MS based sequencing [@PMID: 20113005; @PMID: 25823410].
+This makes it more challenging to identify protein isoforms and other amino acid modifications.
+
 
 3. theoretical studies of proteolysis enzymes [@DOI:https://doi.org/10.1155/2014/960902]
 4. Challenges associated with alternative enzyme choices (non-specific and semi-specific enzymes)
-5. Alternative enzyme choices (one paragraph each?) - LysC
-6. GluC
-7. AspN
-8. Alpha-lytic protease [@DOI:https://doi.org/10.1074/mcp.m113.034710] and how it enables mapping human SUMO sites [@PMID:29079793].
-9. others?
+5. Alternative enzyme choices (one paragraph each?) - 
+The enzyme choice mostly depends on the application.
+In general, for a mere protein identification mostly trypsin is the choice due to the reasons aforementioned.
+However, alternative enzymes are used when the sequence information of a particular protein is limited in the database repositories. 
+Especially, they serve as an advantage in improving the confidence of protein identification [@PMID:12643544; @PMID:20113005].
```

Comment:
These refs suggest that multiple proteases increase sequence coverage not identification confidence, is there evidence that they can improve confidence? please clarify or edit to say improving sequence coverage

Reply:
Has restructured the sentence by saying that using alternative enzymes improves sequence coverage information.